File: Armor.cs
public class Armor {

	public LocationType hitLocation { get; set; }
	public int damageResist { get; set; }
	public bool burningProtection { get; set; }
	public bool corrosionProtection { get; set; }
	public bool crushingProtection { get; set; }
	public bool cuttingProtection { get; set; }
	public bool impalingProtection { get; set; }
	public bool piercingProtection { get; set; }
	public bool toxicProtection { get; set; }
	public bool flexible { get; set; }

	public Armor() {
		LocationType = LocationType.Torso;
		damageResist = 1;
		burningProtection = corrosionProtection = crushingProtection = cuttingProtection = impalingProtection = piercingProtection = toxicProtection = false;
	}

	public Armor(XmlNode node) {
		hitLocation = node.SelectSingleNode("HitLocaton").InnerText;
		damageResist = int.Parse(node.SelectSingleNode("DamageResist").InnerText);
		burningProtection = bool.Parse(node.SelectSingleNode("BurningProtection").InnerText);
		corrosionProtection = bool.Parse(node.SelectSingleNode("CorrosionProtection").InnerText);
		crushingProtection = bool.Parse(node.SelectSingleNode("CrushingProtection").InnerText);
		cuttingProtection = bool.Parse(node.SelectSingleNode("CuttingProtection").InnerText);
		impalingProtection = bool.Parse(node.SelectSingleNode("ImpalingProtection").InnerText);
		piercingProtection = bool.Parse(node.SelectSingleNode("PiercingProtection").InnerText);
		toxicProtection = bool.Parse(node.SelectSingleNode("ToxicProtection").InnerText);
		flexible = bool.Parse(node.SelectSingleNode("Flexible").InnerText);
	}

}

public enum LocationType {
	Suit, Torso, Arms, Legs, Hands, Feet, LeftHand, RightHand, Groin, Head, Skull, Face, Eyes, Neck
}

File: com/twilightparadox/triforce700/core/equipment/Armor.java
package com.twilightparadox.triforce700.core.equipment;

public class Armor {

	public LocationType hitLocation;
	public int damageResist;
	public boolean burningProtection;
	public boolean corrosionProtection;
	public boolean crushingProtection;
	public boolean cuttingProtection;
	public boolean impalingProtection;
	public boolean piercingProtection;
	public boolean toxicProtection;
	public boolean flexible;

	public Armor() {
		hitLocation = LocationType.Torso;
		damageResist = 1;
		burningProtection = false;
		corrosionProtection = false;
		crushingProtection = false;
		cuttingProtection = false;
		impalingProtection = false;
		piercingProtection = false;
		toxicProtection = false;
	}
	
	public LocationType getLocationType() { return hitLocation; }
	public void setLocationType(LocationType hitLocation) { this.hitLocation = hitLocation; }
	
	public int getDamageResist() { return damageResist; }
	public void setDamageResist(int damageResist) { this.damageResist = damageResist; }

	public boolean isBurningProtection() { return burningProtection; }
	public void setBurningProtection(boolean burningProtection) { this.burningProtection = burningProtection; }

	public boolean isCorrosionProtection() { return corrosionProtection; }
	public void setCorrosionProtection(boolean corrosionProtection) { this.corrosionProtection = corrosionProtection; }

	public boolean isCrushingProtection() {return crushingProtection; }
	public void setCrushingProtection(boolean crushingProtection) { this.crushingProtection = crushingProtection; }

	public boolean isCuttingProtection() { return cuttingProtection; }
	public void setCuttingProtection(boolean cuttingProtection) { this.cuttingProtection = cuttingProtection; }

	public boolean isImpalingProtection() { return impalingProtection; }
	public void setImpalingProtection(boolean impalingProtection) { this.impalingProtection = impalingProtection; }

	public boolean isPiercingProtection() { return piercingProtection; }
	public void setPiercingProtection(boolean piercingProtection) { this.piercingProtection = piercingProtection; }

	public boolean isToxicProtection() { return toxicProtection;}
	public void setToxicProtection(boolean toxicProtection) { this.toxicProtection = toxicProtection; }

	public boolean isFlexible() { return flexible; }
	public void setFlexible(boolean flexible) { this.flexible = flexible; }

}

enum LocationType {
	Suit, Torso, Arms, Legs, Hands, Feet, LeftHand, RightHand, Groin, Head, Skull, Face, Eyes, Neck
}